getting-started/forge-addon/forge-addon.adoc

```diff
@@ -0,0 +1,62 @@
+= JBoss Forge Addon
+
+JBoss Forge is a project management tool that extends your IDE by providing wizards and offering a set of extensions (addons) for different technologies. Try JBoss Forge by downloading it at http://forge.jboss.org.
+
+The Wildfly Swarm team provides an addon that eases the user experience on Swarm. 
+
+== Installation
+
+=== From Forge CLI:
+
+[source,shell]
+----
+addon-install-from-git --url https://github.com/forge/wildfly-swarm-addon.git
+----
+
+=== From Eclipse/JBoss Developer Studio: 
+
+- Press Ctrl (Or Command if you are on OSX) + 4 and select the `Install an Addon from GIT` option.
+- Enter `https://github.com/forge/wildfly-swarm-addon.git` as the url and press "Finish"
+
+=== From IntelliJ IDEA: 
+
+- Press Ctrl (Or Command if you are on OSX) + Alt + 4 and select the `Install an Addon from GIT` option.
+- Enter `https://github.com/forge/wildfly-swarm-addon.git` as the url and press "Finish"
+
+=== From Netbeans: 
+
+- Go to the Quick Search field in the upper right corner and type `Install an Addon from GIT`.
+- Enter `https://github.com/forge/wildfly-swarm-addon.git` as the url and press "Finish"
+
+
+== Features
+Wildfly-Swarm Setup command:: 
+Using the Forge shell, navigate to a Java EE project, or create one using the `project-new --named demo --stack JAVA_EE_7` command and type the following command if you're in shell. 
++
+[source,java]
+----
+wildfly-swarm-setup
+----
++
+It should read `Wildfly-Swarm: Setup` if you're in a GUI environment:  
++
+This will make the `wildfly-swarm-maven` plugin to be installed in your project and the Wildfly Swarm BOM added to the `<dependencyManagement>` section. 
++
+You may also find some options that will be added to the Wildfly Swarm maven plugin like the context path, the HTTP port and the HTTP Port offset
+
```

Comment:
Don't you want to mention as well that once the swarm setup has been done, any facet installed will automatically install the related swarm fraction ?